openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1625,31 +1625,176 @@ TBD
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: unsupported_credential_format
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: invalid_proof
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
+
+* Name: invalid_encryption_parameters
+* Usage Location: resource access error response
+* Protocol Extension: OpenID for Verifiable Credentials Issuance
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential-request-errors) of this specification
 
 ## Well-Known URI Registry
 
-This specification registers the well-known URI defined in (#credential-issuer-wellknown) in the IANA Well-Known URI registry defined in RFC 5785 [@!RFC5785].
+This specification registers the following well-known URI in the IANA "Well-Known URI" registry established by [@!RFC5785].
 
 * URI suffix: openid-credential-issuer
-* Change controller: AB/Connect Working Group - openid-specs-ab@lists.openid.net
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Specification document: (#credential-issuer-wellknown) of this document
 * Related information: (none)
 
+## Media Types Registry
+
+This specification registers the following media types in the IANA "Media Types" registry [@!IANA.MediaTypes] in the manner described in [@!RFC6838].
+
+* Type name: `application`
+* Subtype name: `openid4vci-proof+jwt`
+* Required parameters: n/a
+* Optional parameters: n/a
+* Encoding considerations: Uses JWS Compact Serialization, as specified in [@!RFC7515].
+* Security considerations: See the Security Considerations in [@!RFC7519].
+* Interoperability considerations: n/a
+* Published specification: (#jwt-proof-type) of this specification
+* Applications that use this media type: Applications that issue and store verifiable credentials
+* Additional information:
+  - Magic number(s): n/a
+  - File extension(s): n/a
+  - Macintosh file type code(s): n/a
+* Person & email address to contact for further information: TBD
```

Comment:
TBD? Why not the draft authors or the WG?

Reply:
+1 
```suggestion
* Person & email address to contact for further information: Torsten Lodderstedt, torsten@lodderstedt.net
```